pyproject.toml

```diff
@@ -25,15 +25,13 @@ dependencies = [
     "numpy",
     "pandas",
     "xarray",
+    "xcube>=1.5.1",
```

Comment:
@TejasMorbagal: I think xcube should be also stated in the dependencies of the `pyproject.toml`. But there is a comment above, that xcube-core should be added later. Can you please clarify? Thanks

Reply:
@konstntokas The dependencies listed here are installed by default from PyPI and xcube has the name xcube-core in PyPI.

Reply:
Shall I include `"xcube-core>=1.5.1"` then?

Reply:
The issue is there is no 1.5.1 release done on PyPI, the only release I see is 1.4.1. We mainly distribute xcube and it's plugin as conda packages. But we need to automate the release on PyPI as well. For now, we shouldn't list it here.

Reply:
Ok I removed it. Thanks